app/views/govuk_component/docs/metadata.yml

```diff
@@ -17,6 +17,15 @@ fixtures:
     from:
```

Comment:
Sorry, not sure which lines you meant. If I understand the file structure correctly, lines 17 and 20 start separate examples in the component guide. Lines 17 to 19 was an existing one, I added lines 20 - 28 as a test of the new expand/collapse when applied to the 'from' field.